CrisisTextLine/Elmer
# This branch contains the most recent sample data. If we merge this in…

#### What's this PR do?
Provides the most recent, robust version of the test/sampleData file, which includes lots more data, and more varied data, to help in testing. Once we merge it in to master, it should also hopefully help prevent common merge conflicts we're experiencing solely based on there being a lot of different versions of the sampleData file out there.
#### Where should the reviewer start?
test/sampleData.js
#### How should this be manually tested?
Try it out with any tests you're working on and make sure the data set works for those tests.
#### Any background context you want to provide?
#### What are the relevant tickets?
None. Just got tired of dem sampleData merge conflicts.
#### Questions:

…to master we may avoid a lot of merge conflicts based solely on the sampleData file down the road.
